src/components/TracksTable.vue

```diff
@@ -50,9 +50,16 @@
         <span v-if="props.value">
           {{ props.value | length }}
         </span>
-        <span v-else class="red--text">
-          --:--
-        </span>
+        <VTooltip v-else bottom>
+          <template v-slot:activator="{ on }">
+            <span v-on="on" class="red--text">
```

Comment:
The `--:--` in red text looks a bit weird to me, I think making the icon red is enough.